gui/webdisplay/test/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@
 
 
 # test only can be run if Firefox or Chrome are detected on the system
-if (CHROME_EXECUTABLE OR FIREFOX_EXECUTABLE)
+if((CHROME_EXECUTABLE OR FIREFOX_EXECUTABLE) AND MACOSX_VERSION VERSION_GREATER_EQUAL 13)
```

Comment:
I am confused ... doesn't that disable the test on *ALL* platform except for MacOS 14+?  Is that the intent?